packages/terra-avatar/docs/upgrade-guide.md

```diff
@@ -1,4 +1,36 @@
 # terra-avatar Upgrade Guide
+## Changes from version 2 to version 3
+
+### Removed
+* `Shared User` subcomponent From Avatar
+
+### Added
+* the `generic` subcomponent that replaces the `sharedUser` subcomponent with a new `variant` prop that can be `single-user`, `shared-user`, or `provider`.
+* `variant` prop this will take values for sub-variants `single-user`, `shared-user` and `provider`.
+
+### Changes to CSS Custom Properties
```

Comment:
Do not include changes to css custom properties in the upgrade guide. This is an implementation detail now.

Reply:
should we add this to README.md..?